Add pt-BR locale

guieiras-dev opened the pull request:
I've followed the instructions at [src/locale/README.md](https://github.com/crankyoldgit/IRremoteESP8266/blob/master/src/locale/README.md), and here is my PR with pt-BR locale.

I did not find instructions on how to test it. So, if anything is broken, let me know and I will fix it as soon as possible.

guieiras committed:
pt-BR: add Locale

guieiras committed:
Fixed missing translations

crankyoldgit commented on src/locale/pt-BR.h:
```diff
@@ -0,0 +1,167 @@
+    "Edite e aumente o valor de `kCaptureBufferSize`."
+#endif  // D_WARN_BUFFERFULL
+
+#endif  // LOCALE_DEFAULTS_H_
```
From the linter:
`src/locale/pt-BR.h:167:  #endif line should be "#endif  // LOCALE_PT_BR_H_"  [build/header_guard] [5]`

I'll do a commit to fix this and add a few other tests too.

crankyoldgit committed:
Fix linter error

* src/locale/pt-BR.h:167:  #endif line should be "#endif  // LOCALE_PT_BR_H_"  [build/header_guard] [5]

crankyoldgit committed:
Correct copyright date.

crankyoldgit committed:
IRrecvDumpV2: Portuguese/Brazilian build option

Add easy support for PlatformIO to build the example with Brazilian Portuguese.
This will also be used Travis CI build so it should pick up any errors.

crankyoldgit committed:
Fix linter error I just introduced

... to fix a different linter error. Sigh.

crankyoldgit reviewed (approved):
Many many thanks. Great first contribution to the library!
You've more than earned your 1/4 of a Hacktoberfest T-shirt here!

guieiras-dev commented:
Awesome! It was a pleasure to colaborate.

Thanks a lot.

crankyoldgit commented:
FYI, the changes mentioned above have now been included in the new [v2.7.12 release](https://github.com/crankyoldgit/IRremoteESP8266/releases/tag/v2.7.12) of the library.
